src/db/index.js

```diff
@@ -20,19 +22,68 @@ sequelize
     console.error('Unable to connect to the database:', err);
   });
 
-const Buy = sequelize.define('buy', orderSchema);
-const Sell = sequelize.define('sell', orderSchema);
+const Buy = sequelize.define('buy', orderSchema, {
+  indexes: [ // A BTREE index with a ordered field
+    {
+      method: 'BTREE',
+      fields: ['price']
+    },
+    {
+      method: 'BTREE',
+      fields: ['createdAt']
+    }
+  ]
+});
+const Sell = sequelize.define('sell', orderSchema, {
+  indexes: [ // A BTREE index with a ordered field
+    {
+      method: 'BTREE',
+      fields: ['price']
+    },
+    {
+      method: 'BTREE',
+      fields: ['createdAt']
+    }
+  ]
+});
 const Pair = sequelize.define('pair', pairSchema);
 
-Pair.sync().then(result => console.log(result));
+Pair
+  .sync()
+  .then(result => console.log(result))
+  .then(() => Pair.findAll())
+  .then(([{ dataValues }]) => console.log(dataValues));
 
-Buy.belongsTo(Pair, {as: 'pair'});
+Buy.belongsTo(Pair, { as: 'pair' });
 Pair.hasMany(Buy);
-Sell.belongsTo(Pair, {as: 'pair'});
+Sell.belongsTo(Pair, { as: 'pair' });
 Pair.hasMany(Sell);
 
-Buy.sync().then(result => console.log('Buy Sync', result));
-Sell.sync().then(result => console.log('Sell Sync', result));
+//TEST CORE QUERY:
+const sortBuys = () => {
+  console.log('starting sort');
+  Buy
+    .max('price')
+    .then(max => Buy.findAll({ where: { price: max }, order: [[sequelize.col('createdAt'), 'ASC']]}))
```

Comment:
DESC

Reply:
I was mistaken earlier... it needed to be DESC if it ordered by price, but since I'm ordering by 'createdAt' the goal is to obtain the earliest order at a given price first. Therefore, 'ASC'